watertap/tools/oli_api/util/fixed_keys_dict.py

```diff
@@ -45,50 +88,55 @@ def pprint(self):
             print(f" {key}\n - {value}\n")
 
 
+input_unit_set_temp = {
+    "molecularConcentration": {
+        "oli_unit": "mg/L",
+        "pyomo_unit": pyunits.mg / pyunits.L,
+    },
+    "mass": {"oli_unit": "mg", "pyomo_unit": pyunits.mg},
+    "temperature": {"oli_unit": "K", "pyomo_unit": pyunits.K},
+    "pressure": {"oli_unit": "Pa", "pyomo_unit": pyunits.Pa},
+    "enthalpy": {"oli_unit": "J", "pyomo_unit": pyunits.J},
+    "vaporAmountMoles": {"oli_unit": "mol", "pyomo_unit": pyunits.mol},
+    "vaporMolFrac": {
+        "oli_unit": "mol/mol",
+        "pyomo_unit": pyunits.mol / pyunits.mol,
+    },
+    "totalVolume": {"oli_unit": "L", "pyomo_unit": pyunits.L},
+    "pipeDiameter": {"oli_unit": "m", "pyomo_unit": pyunits.meter},
+    "pipeFlowVelocity": {
+        "oli_unit": "m/s",
+        "pyomo_unit": pyunits.meter / pyunits.second,
+    },
+    "diskDiameter": {"oli_unit": "m", "pyomo_unit": pyunits.meter},
+    "diskRotatingSpeed": {"oli_unit": "cycle/s", "pyomo_unit": 1 / pyunits.second},
+    "rotorDiameter": {"oli_unit": "m", "pyomo_unit": pyunits.meter},
+    "rotorRotation": {"oli_unit": "cycle/s", "pyomo_unit": 1 / pyunits.second},
+    "shearStress": {"oli_unit": "Pa", "pyomo_unit": pyunits.Pa},
+    "pipeDiameter": {"oli_unit": "m", "pyomo_unit": pyunits.meter},
+    "pipeRoughness": {"oli_unit": "m", "pyomo_unit": pyunits.meter},
+    "liquidFlowInPipe": {
+        "oli_unit": "L/s",
+        "pyomo_unit": pyunits.L / pyunits.second,
+    },
+    "gasFlowInPipe": {"oli_unit": "L/s", "pyomo_unit": pyunits.L / pyunits.second},
+    "viscAbs2ndLiq": {
+        "oli_unit": "Pa-s",
+        "pyomo_unit": pyunits.Pa * pyunits.second,
+    },
+    "alkalinity": {"oli_unit": "mg HCO3/L", "pyomo_unit": pyunits.mg / pyunits.L},
```

Comment:
This example, among others, is why Paul and I gave up on an automatic approach. It seemed like you could be working forever on every edge case and still miss some.

Reply:
Right- we'd need to set valid units accepted for each property, which is documented. We may catch bugs along the way (i.e., what's documented may not match reality in some cases).